Xamarin.PropertyEditing.Mac/Controls/PredefinedValuesEditor.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections;
+using System.Diagnostics;
+using System.ComponentModel;
+
+using Foundation;
+using AppKit;
+using CoreGraphics;
+
+using Xamarin.PropertyEditing.ViewModels;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	internal class PredefinedValuesEditor<T>
+		: PropertyEditorControl
+	{
+		public PredefinedValuesEditor ()
+		{
+			base.TranslatesAutoresizingMaskIntoConstraints = false;
```

Comment:
The 'traditional' approach is that a control doesn't set it's own `TranslatesAutoresizingMaskIntoConstraints` value. The parent set's it on it's children *if* that parent control attaches constraints to the children. By setting `base.TranslatesAutoresizingMaskIntoConstraints` here you're assuming that every container this control is placed inside *will* use constraints to size/position this control.